roles/kubernetes-apps/kubelet-csr-approver/meta/main.yml

```diff
@@ -13,7 +13,8 @@ dependencies:
         namespace: "{{ kubelet_csr_approver_namespace }}"
         chart_ref: "{{ kubelet_csr_approver_chart_ref }}"
         chart_version: "{{ kubelet_csr_approver_chart_version }}"
-        wait: true
+        wait: "{{ not (kube_network_plugin == 'cni') }}"
```

Comment:
```suggestion
        wait: "{{ kube_network_plugin != 'cni' }}"
```

Reply:
Certainly! I’ve squashed the fix commits into the main commits as requested, so the final PR is now clean and organized without separate fix commits. Let me know if there’s anything else to adjust.